packages/hash/api/codegen.yml

```diff
@@ -36,7 +36,7 @@ generates:
         DataTypeWithMetadata: "@hashintel/hash-subgraph#DataTypeWithMetadata"
         PropertyTypeWithMetadata: "@hashintel/hash-subgraph#PropertyTypeWithMetadata"
         EntityTypeWithMetadata: "@hashintel/hash-subgraph#EntityTypeWithMetadata"
-        EntityWithMetadata: "@hashintel/hash-subgraph#EntityWithMetadata"
+        Entity: "@hashintel/hash-subgraph#EntityWithMetadata"
```

Comment:
`@hashintel/hash-subgraph#EntityWithMetadata` is kept the same to prevent GQL name clashes

Reply:
I didn't consider we define Entity as a scalar now. Complete rename in 5146d27288ef5281285dc6576ff12cad69c8d555